src/main/java/com/google/api/codegen/transformer/SampleTransformer.java

```diff
@@ -340,9 +357,28 @@ private InitCodeContext createInitCodeContext(
               .identifier(identifier)
               .readFromFile(attr.getReadFile())
               .sampleArgumentName(attr.getSampleArgumentName())
+              .description(attr.getDescription())
               .build();
       builder.put(identifier, config);
     }
     return builder.build();
   }
+
+  private ImmutableList<List<String>> paramDocLines(
```

Comment:
Yes.

Reply:
Could you add the function comment?
And as per the comment above, could you pass the line length as a param here, so that the constant is only used in one function rather than two?

Reply:
Sorry forgot to push the local changes after commenting above :/ It will come right away.